week-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,61 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "./assets/cute-cat-a.jpg",
+  "./assets/cute-cat-b.jpg",
+  "./assets/cute-cat-c.jpg",
+  "./assets/cute-cat-d.jpg",
+  "./assets/cute-cat-e.jpg",
+  "./assets/cute-cat-f.jpg",
 ];
 
+const forwardSlide = document.querySelector("#forward-btn");
+const backwardSlide = document.querySelector("#backward-btn");
+const imageSelector = document.querySelector("#carousel-img");
+const stopBtn = document.querySelector("#stop-btn");
+const autoBackBtn = document.querySelector("#auto-back-btn");
+const autoForwardBtn = document.querySelector("#auto-forward-btn");
 
-// Write your code here
\ No newline at end of file
+let num = 0;
+imageSelector.style.width = "600px";
```

Comment:
i would definitely do this in css! It works here, you might just want to keep JS seperate for functionality